docs/csharp/cs-dev-kit-faq.md

```diff
@@ -193,6 +193,21 @@ C# Dev Kit requires that it has built your project successfully before tests wil
 
 Build your solution by right-clicking on the solution in the Solution Explorer and select **Build** or `kb(workbench.action.tasks.build)`. Once the build has been completed, your tests will appear in the Test Explorer panel.
 
+### How do I collect logs for troubleshooting issues with Test Explorer?
+
+If you are encountering issues with Test Explorer, you can enable diagnostic logging to gather more information for troubleshooting:
+1. Increase Test Explorer Verbosity:
+Navigate to the C# Dev Kit settings and increase the Test Explorer Verbosity setting from minimal to diagnostic. This will generate more detailed logs.
```

Comment:
```suggestion
    Navigate to the C# Dev Kit settings and increase the Test Explorer Verbosity setting from `minimal` to `diagnostic`. This will generate more detailed logs.
```